typings/missing.d.ts

```diff
@@ -5,3 +5,4 @@ declare module 'react-file-download';
 declare module 'react-zeroclipboard';
 declare module 'reactableMSK';
 declare module 'redux-seamless-immutable';
```

Comment:
noImplicitAny is very useful and should remain on, which means we need to declare these modules.